scripts/ci/dind-setup.sh

```
@@ -0,0 +1,336 @@
+        docker rm -f "$container_name"
+        return 1
+    fi
+}
+
+# Production service helpers
+_wait_for_prod_ready() {
+    local port=${1:-3000}
+    local timeout=${2:-60}
+    
+    echo "⏳ Waiting for production service to be ready..."
+    for i in $(seq 1 $timeout); do
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)